packages/row/src/react/index.js

```diff
@@ -374,22 +375,27 @@ renderProgress.propTypes = {
 
 // NOTE: the `title` prop clashes with a native html attr so we're exclude it
 //       from being mistakenly used in any child component
-const Row = ({ title, titleTruncated, ...props }) => (
-  <div {...styles.row(props)} {...filterReactProps(props)}>
-    {renderOverlays(props)}
+const Row = ({ title, titleTruncated, ...props }) => {
+  const desktop = useMatchMedia('(min-width: 769px)')
+  if (!props.size) props.size = desktop ? vars.sizes.medium : vars.sizes.small
 
-    <Words image={props.image} size={props.size} actionBar={props.actionBar}>
-      <Title size={props.size} truncated={titleTruncated}>
-        {title}
-      </Title>
+  return (
```

Comment:
what really changed here...?  hard to parse.

Reply:
addition of the `<div {...styles.row(props)} {...filterReactProps(props)}>`?

Reply:
the only change is a new `props.size` based on the media query is added. it can be override if a size is provided